composer.json

```diff
@@ -32,7 +32,7 @@
   "prefer-stable": true,
   "require": {
     "oat-sa/generis": "15.27.0",
-    "oat-sa/tao-core": "53.4.2",
+    "oat-sa/tao-core": "53.6.0",
```

Comment:
Did we check those features included will not cause any other side effects?

Reply:
2 of them - just scripts changes:
* https://github.com/oat-sa/tao-core/pull/3870
* https://github.com/oat-sa/tao-core/pull/3854

The only thing is this one - https://github.com/oat-sa/tao-core/pull/3860

But from my perspective, these changes are not so critical.
Wdyt?

Reply:
@vbyndych @KirylHatalski 
Guys, what do you think?
Can I include your changes in the `2023.09.1` tao-community backport release to avoid creating backports for `tao-core` and `tao-task-queue`?
Or do you think they can somehow affect?

Reply:
Generis fix was not validated by QA. Lets please check with AUT team first about any risk.

Reply:
Why do we bump tao-core with a minor version instead of backporting changes?